src/opnsense/mvc/app/models/OPNsense/DHCP/ACL/ACL.xml

```diff
@@ -0,0 +1,28 @@
+<acl>
+    <page-services-dhcprelay>
```

Comment:
movement seems a bit inconsistent: either move all dhcp related things or just the new MVC pages here